gctscript/modules/ta/indicators/rsi.go

```diff
@@ -0,0 +1,50 @@
+package indicators
+
+import (
+	"fmt"
+
+	objects "github.com/d5/tengo/v2"
+	"github.com/thrasher-corp/go-talib/indicators"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/modules"
+)
+
+// RsiModule relative strength index indicator commands
+var RsiModule = map[string]objects.Object{
+	"rsi": &objects.UserFunction{Name: "rsi", Value: rsi},
+}
+
+func rsi(args ...objects.Object) (objects.Object, error) {
+	if len(args) != 2 {
+		return nil, objects.ErrWrongNumArguments
+	}
+
+	ohlcvInput := objects.ToInterface(args[0])
+	ohlcvData := ohlcvInput.([]interface{})
```

Comment:
possible panic

Reply:
Yeah will add in checks for all these to stop any happy little accidents for people not passing data in from the ohlcv() method